services/pane/publish.js

```diff
@@ -88,7 +107,17 @@ function addErrorsOrWarnings(errors, modifier) {
 
     // add each place where the error/warning occurs
     _.each(error.errors, function (item) {
-      var itemEl = dom.create(`<li><span class="error-label">${item.label}</span>${addPreview(item.preview)}</li>`);
+      let itemEl;
+
+      if (item.fieldName && item.data) {
+        // add a link to open the specific form to update the data
+        itemEl = dom.create(`<li data-error-ref="${item.ref}" data-error-path="${link.getPathFromField(item.fieldName, item.data)}"><span class="error-label error-label-link">${item.label}</span>${addPreview(item.preview)}</li>`);
```

Comment:
Why not use an anchor tag for the link? I know you need multiple pieces of data (`error-path` and `error-ref`) and so one `href` value wouldn't work, but it should be a link semantically right?